### PR TITLE
Update about-quickstart.md

### PR DESCRIPTION
**Pull request by shreyas1599**
Made the changes suggested in #38. Also, a couple of changes I'd like to make as well, but need an opinion before doing so. A few sentences like the following use `will`. I suggest removing or replacing the word. For example:

> It should explain the basic operations that most users will want to perform on the site, or the operations they will perform when they use the API. 

It should explain the basic operations that most users perform on the site, or the operations they perform when they use the API. 

> Start by telling users what the quickstart will do

Start by telling users what the quickstart does. 

Basically, a few tense changes. There are a couple more `will`s in the document. I'll make these changes if(/when) a maintainer accepts them.

### CODE REVIEW

**Comment by shreyas1599** [comment]
@jaredmorgs Can I go ahead and add a commit for the other changes I suggested above?

**Comment by jaredmorgs** [comment]
Yep go for it!

**Comment by jaredmorgs** [comment]
Just that minor change in the last sentence looks good. All other changes look good. I'll merge this in now.
